ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -240,12 +245,13 @@ if have_python3
     install_dir: nnstreamer_libdir
   )
 
-  shared_library('nnstreamer_python3',
+  ### .abi3 See https://www.python.org/dev/peps/pep-3149/, This does not support python < 3.2
+  shared_library('nnstreamer_python.abi3',
     'tensor_filter_python_api.c',
     name_prefix: '',
     dependencies: nnstreamer_filter_python3_deps,
     install: true,
-    install_dir: filter_subplugin_install_dir
+    install_dir: py3.get_install_dir()
```

Comment:
py3.get_install_dir() would return the location of the site-packages directory for the current building host, not the target platform. This looks dangerous.

Reply:
Have you got any suggestion? If not, I think I should revert this change and leave this job to do for debian and rpmbuild.

Reply:
I reverted for the time being as reverted version still resolves #2303.
I think this could be handled in different PR if needed 👍